File: wordcount.js
function countWords(stringOfWords) {
  // your code here
   var stringcount = {}
    var words = stringOfWords.split(' ')
  if (stringOfWords ==""){
    return {};
  }

    for(var x = 0; x < words.length; x ++)
    {
      if (stringcount[words[x]] == undefined)
      { 
        stringcount[words[x]] = 1
  
      }
      else
      {
        stringcount[words[x]]++;
      }
    console.log("result", stringcount)
    
    }
  return stringcount;
}

var result1 = countWords('ask a bunch get a bunch');
console.log('should log "{ask: 1, a: 2, bunch: 2, get: 1}":', result1);

var result2 = countWords('');
console.log('should log "{}":', result2);

File: wordcount.py
def countWords(stringOfWords):
    stringcount = {}
    words = stringOfWords.split(' ')
    print(words)
    if (stringOfWords ==""):
        return {}
    i = 0
    for x in words:
      if words[i] in stringcount:
        stringcount[words[i]] = stringcount[words[i]] + 1
        i = i +1
      
      else:
        stringcount[words[i]] = 1
        i = i +1
    return stringcount


result1 = countWords('ask a bunch get a bunch')
print('should log "{ask: 1, a: 2, bunch: 2, get: 1}":', result1)

result2 = countWords('')
print('should log "{}":', result2)
